WalletWasabi/Legal/Assets/LegalDocumentsGingerWallet.txt

```diff
@@ -0,0 +1,484 @@
+
+
+    It is Your sole responsibility to keep Your wallet backups, Your wallet files, and Your passwords secure. You should not provide this information to anyone, including any Service Provider representative. Encrypted private key information is stored locally on Your computer in a wallet file. Private keys can be accessed with the password, which You created at the generation of the wallet.
+
+    IF YOU FORGET OR LOSE YOUR BACKUP, YOU WILL NEVER BE ABLE TO RECOVER ANY BITCOIN IN YOUR WALLET, AND YOU WILL SUFFER A COMPLETE, IRRECOVERABLE LOSS OF ALL BITCOIN IN YOUR WALLET. IN THE EVENT OF SUCH LOSS, WE SHALL NOT BE LIABLE FOR ANY LACK OF ACCESS TO THE WALLET AND WE SHALL NOT BE OBLIGED TO MAKE WALLET ACCESS, OR THE KEYS REQUIRED FOR THAT PURPOSE AVAILABLE.
+
+    The Service Provider has no responsibility and will not be liable for any loss or damage You suffer from the loss or misappropriation of Your backup.
+
+    From the point of downloading, We do not store or manage any data, and We have no knowledge or information about the installation or the use of the wallet.
+
+
+1.4 PRIVACY POLICY
+
+    Our Privacy Policy is expressly incorporated into these Terms and should be read carefully as it contains important information about handling personal data. Our Privacy Policy may be found in Chapter II, below.
+
+
+1.5 CHANGES TO THESE TERMS
+
+    We may amend these Terms, including when there are changes to Our Services, technology, or any law or regulation to which We are subject and/or for any other reason. If We do, We will post the updated Terms on Our website; however, for any material changes required to these Terms, We will ask You to read through them and We will request Your re-acceptance. Please refer to the “Last Updated” date and Version Number above.
+
+
+   Any amended Terms will become effective on the day that they are posted (Last Updated) and will apply prospectively to the use of Our Services after such changes become effective, except that any changes addressing new functions of Our Services or changes made for legal reasons will come into effect immediately. Your continued use of our Services following the effective date of such changes will constitute Your acceptance of such changes. If You do not agree to any amended Terms, You must discontinue using Our Services, otherwise, the liability clauses set out in these Terms will apply.
+
+
+2 THE COINJOIN COORDINATOR SERVICES
+
+===================================
+
+
+2.1 DESCRIPTION OF SERVICES
+
+    This Software consists of the Client Application and the CoinJoin Coordinator (both as defined below) and function as a free, open-source, non-custodial desktop wallet. The Software does not constitute an account where We or other third parties serve as financial intermediaries or custodians of Your Bitcoins.
+
+The Software and Our Services are therefore not a bank, a custodian, an exchange, a financial intermediary, or a regulated financial institution.
```

Comment:
Would it make sense to mention that it is also not a money transmitting service?

Reply:
L144 covers it.